openhtf/util/configuration.py

```diff
@@ -46,6 +46,7 @@
     'This value will override any existing config value at config load time '
     'and will be a string')
 
+_LOG = logging.getLogger('HTFTest')
```

Comment:
Things in util/ aren't really core framework, and probably should use either a different logger, or just the root logger.  That is, util/configuration.py is actually a standalone utility file, not intended for use only by OpenHTF (though it's unlikely it'll actually be used by anything else).